packages/esm-appointments-app/src/config-schema.ts

```diff
@@ -90,6 +90,30 @@ export const configSchema = {
     _description: 'Whether to allow scheduling of all-day appointments (vs appointments with start time and end time)',
     _default: true,
   },
+  checkInButton: {
+    enabled: {
+      _type: Type.Boolean,
+      _description: 'Whether the check-in button on the "Appointments" list should be enabled',
+      _default: true,
+    },
+    customUrl: {
```

Comment:
I haven't implemented this custom URL functionality yet, but hope to have a separate ticket and PR out for that today or tomorrow... if we decided we don't want/can't do this, i will remove this parameter and the one below.